docker-compose/docker-compose.multiserver.mariadb.yml

```diff
@@ -107,4 +114,4 @@ services:
       - ACTIVEMQ_BROKER_USERNAME=admin
       - ACTIVEMQ_BROKER_PASSWORD=password
     volumes:
-      - data:/data
+      - data:/data
```

Comment:
Please fix no newline error. [A single newline should end the file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file/18789#18789).

Reply:
Ready